### ISSUE TITLE
Ideas for Improvement

### ISSUE THREAD

**Issue by isteiger** [reedycreektimes/reedycreektimes.github.io#1]
Hi! I'm Ian Steiger, a former student at RCMMS, and I stumbled upon your repo. Right now, I see that you are making a new HTML file for each article. I know some ways to improve your workflow so this is less of a mess to make. A Jeykll blog where there is a new Markdown file for each article which is then compiled to HTML automatically by GitHub Pages is one. You can still have a custom theme and layout for this. [Mozilla's old Release Management blog](https://release.mozilla.org/) is a good example of one. In [their GitHub repo](https://github.com/mozilla/release-blog), you can see how it is structured. Another idea of improving this that I have is a database (a free one like Firebase's Realtime Database or Firestore) where the articles are stored there and then dynamically loaded into the site by JavaScript. If you would like my help with any of the development you are doing, let me know. My Discord is isteiger11#7458 or my school email is iksteiger@students.wcpss.net.

P.S you may know me by [the Dreambox petition](bit.ly/stopdreambox) I made in 7th grade, or the old [Wake Student's site](https://main-wakestudents.netlify.app/) (formerly Reedy Creek Students' site) that I made along with another former student @asavage7 (Alex Savage). Currently, I am working on [the Pinkerton Minecraft Server Network](https://www.pinkertonmc.com).

**Comment by LockeShor** [comment]
I agree with Ian, and something else is that alongside using Markdown as opposed to raw HTML for each page; something you could use is a [google doc to markdown addon ](https://workspace.google.com/marketplace/app/docs_to_markdown/700168918607)to be able to edit in a familiar workspace and then export to the site.

**Comment by sc-delll** [comment]
yea integrating it is the hard part
